test-files/cmd/glacial/NonDestructiveConflationTest.sh

```diff
@@ -25,17 +26,46 @@ hoot changeset-derive --stats \
 # |  REF1  | 8126 |        1 |  82 |
 # |  REF2  | 1032 |        0 | 218 |
 #
-# @b2ef060
+# @b2ef060 - Baseline
 #
 # |        | Node | Relation | Way |
 # | Create |  856 |        0 | 900 |
 # | Delete |    2 |        1 |  59 |
 # | Modify |    0 |        0 |   4 |
 #
-# @bb953fb
+# @bb953fb - WaySplitter keep IDs
 #
 # |        | Node | Relation | Way |
 # | Create |  859 |        0 | 860 |
 # | Delete |    1 |        1 |  17 |
 # | Modify |    0 |        0 |  45 |
 #
+# @ - WayJoiner checking tags
+#
+# |        | Node | Relation | Way |
+# | Create |  859 |        0 | 670 |
+# | Delete |    1 |        1 |  17 |
+# | Modify |    0 |        0 |  33 |
+#
+# @ - WayJoiner no tag checking
+#
+# |        | Node | Relation | Way |
+# | Create |  859 |        0 | 474 |
+# | Delete |    1 |        1 |  17 |
+# | Modify |    0 |        0 |  10 |
+#
+# @ - WayJoiner rejoin siblings
+#
+# |        | Node | Relation | Way | 
+# | Create |  859 |        0 | 312 | 
+# | Delete |    1 |        1 |  18 | 
+# | Modify |    0 |        0 |   9 | 
+#
+# @ - WayJoiner join at node
+#
+# |        | Node | Relation | Way |
+# | Create |  859 |        0 | 309 |
+# | Delete |    1 |        1 |  17 |
+# | Modify |    0 |        0 |   9 |
+
+
```

Comment:
900 new ways down to 309 is pretty good progress.  Especially since there are 82 original ways and adding in 218 more ways.  We delete less ways and modify more of them.